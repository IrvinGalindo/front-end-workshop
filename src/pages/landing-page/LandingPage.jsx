import React from 'react';

import { MainLayout } from '../../layouts';
import { Cover, ListPets } from '../../components';

import './landingPage.scss';

const pets = [
  {
    id: 'abc',
    img: 'https://duaw26jehqd4r.cloudfront.net/items/1M0C3r3X2t0x2q342A2A/benito.png?v=933367f2',
    name: 'Benito',
    details:
      'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ad eum dolorum architecto obcaecati enim dicta praesentium, quam nobis! Neque ad aliquam facilis numquam. Veritatis, sit.',
  },
  {
    id: 'def',
    img: 'https://duaw26jehqd4r.cloudfront.net/items/0O3b091L2O3N3x1l2Q3Q/remmy.png?v=69c651fc',
    name: 'Remmy',
    details:
      'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ad eum dolorum architecto obcaecati enim dicta praesentium, quam nobis! Neque ad aliquam facilis numquam. Veritatis, sit.',
  },
];

const LandingPage = () => (
  <MainLayout>
    <Cover />
    <section className="yta_Landing-recentPets">
      <h2 className="yta_Landing-recentPets-title">Busco un hogar Â¿me adoptas?</h2>
      <ListPets pets={pets} />
    </section>
  </MainLayout>
);

export default LandingPage;
