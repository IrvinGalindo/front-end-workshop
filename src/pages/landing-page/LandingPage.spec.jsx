import React from 'react';
import { shallow } from 'enzyme';

import LandingPage from './LandingPage';

describe('LandingPage', () => {
  const landingPage = shallow(<LandingPage />);

  it('should render', () => {
    expect(landingPage).toMatchSnapshot();
  });
});
