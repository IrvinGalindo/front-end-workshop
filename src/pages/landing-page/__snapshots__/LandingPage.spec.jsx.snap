// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage should render 1`] = `
<MainLayout>
  <Cover />
  <section
    className="yta_Landing-recentPets"
  >
    <h2
      className="yta_Landing-recentPets-title"
    >
      Busco un hogar Â¿me adoptas?
    </h2>
    <ListPets
      pets={
        Array [
          Object {
            "details": "Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ad eum dolorum architecto obcaecati enim dicta praesentium, quam nobis! Neque ad aliquam facilis numquam. Veritatis, sit.",
            "id": "abc",
            "img": "https://duaw26jehqd4r.cloudfront.net/items/1M0C3r3X2t0x2q342A2A/benito.png?v=933367f2",
            "name": "Benito",
          },
          Object {
            "details": "Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ad eum dolorum architecto obcaecati enim dicta praesentium, quam nobis! Neque ad aliquam facilis numquam. Veritatis, sit.",
            "id": "def",
            "img": "https://duaw26jehqd4r.cloudfront.net/items/0O3b091L2O3N3x1l2Q3Q/remmy.png?v=69c651fc",
            "name": "Remmy",
          },
        ]
      }
    />
  </section>
</MainLayout>
`;
